Towny/src/main/java/com/palmergames/util/StringMgmt.java

```diff
@@ -218,12 +218,9 @@ public static boolean isAllUpperCase(@NotNull String string) {
 		if (string.isEmpty())
 			return false;
 
-		char underscore = '_';
 		for (int i = 0; i < string.length(); i++) {
 			char character = string.charAt(i);
-			if (character == underscore)
-				continue;
-			if (!Character.isUpperCase(character))
+			if (Character.isLowerCase(character))

```

Comment:
I believe this will break when fed an underscore, no? They are considered lowercase.

Reply:
Nope, underscores are neither considered uppercase nor lowercase so it'll work